packages/components/package.json

```diff
@@ -62,6 +63,7 @@
     "reactour": "^1.13.4",
     "simulant": "^0.2.2",
     "styled-components": "^4.1.3",
+    "timezone-support": "^2.0.2",
```

Comment:
This give us list of supported timezones, so we can check if specified timezone is valid.